Page for tools (ACCESS-NRI releases) (#1093)
ACCESS-Hive/access-hive.github.io: docs/about/release_list.md

[Hive Forum]: https://forum.access-hive.org.au
# ACCESS-NRI supported releases

<div class="card-container">
    <a href="/about/releases" class="horizontal-card">
        <div class="card-image-container">
            <img class="img-contain white-background" src="/assets/rocket_release.svg" alt="ACCESS-NRI release stages">
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-NRI Release Stages</span>
            <span>
                Information around release stages terminology and support which help give a general indication on the state of the product.
            </span>
        </div>
    </a>
</div>

## Release type

> The types used to categorise the release product are broad. Not all releases fall neatly into these types. Other releases include training and explainer videos on [_YouTube_](https://www.youtube.com/@AustraliasClimateSimulator), and visualisations on [_Vimeo_](https://vimeo.com/accessnri) and may have specific uses.

Author: Yes, thanks, I thought it would be good to mention some things that oculd be considered 'release' but doesn't make sense to include here as they might be a specific project or one off and there could be other things i'm missing too.

:octicons-ai-model-16: [**Model**](#model): Refers to climate model components and [configurations](/about/releases/#fn:1). <br>
:material-tools: [**Tool**](#tool): Refers to supporting tools which can be pre and post processing, evaluation and diagnostics. <br>
:material-database: [**Data**](#data): Collections listed are curated collections which can include data required for model runs, observations for evaluation and some model outputs.

> Links in the tables below are release notes that are mostly posted on the [Hive Forum]. 

Reviewer: ```suggestion
Links in the tables below point to release notes that are usually posted on the [Hive Forum]. 
```

> In these topics scroll or drag the slider to the latest post for most recent updates.

Reviewer: ```suggestion
In these release notes, make sure you scroll down to the latest post for most recent updates.
```

|[Regional nesting suite (rAM3)](https://forum.access-hive.org.au/t/access-ram3-release-information/4308) | An implementation of the UK Met Office (UKMO) regional nesting suite that supports creating regional atmosphere/land configurations in an Australian context. | Model |
|[Model tools](https://forum.access-hive.org.au/t/model-tools/4696) |There are a number of tools deployed on _Gadi_ to support model tasks. FRE-NCtools, mppnccombine-fast, esmf. | Tool |
|[Ancillary Data](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f9243_2030_4580_5589) | This collection provides access to replica, ancillary and other data that is considered of value to ACCESS-NRI and the broader community. | Data |
|[ACCESS Model Scaling](https://forum.access-hive.org.au/t/access-nri-model-scaling-repository-a-collection-of-parallel-scalability-studies/5426) | A collection of Jupyter Notebooks that generate and display scaling data for ACCESS-NRI models. | Tool |

> </div> -->

Reviewer: I presume this was a previous attempt at having one single table with all releases. 
I like the current structure with multiple tables related to each type. 

I would simply delete this commented portion.

```suggestion
```

Author: yep left over from before

### Model
| |  |
| :--- | :---------- |

> |[CABLE](https://forum.access-hive.org.au/t/cable-is-now-under-git-and-github/1643) | Land surface model in ACCESS models. Code available on GitHub. |

Reviewer: ```suggestion
|[CABLE](https://forum.access-hive.org.au/t/cable-is-now-under-git-and-github/1643) | Land surface model in ACCESS models. Code available on GitHub |
```

|[ACCESS-OM2](https://forum.access-hive.org.au/t/access-om2-release-information/1602) | Global coupled Ocean-Sea Ice Model developed by COSIMA |
| [ACCESS-ESM1.5](https://forum.access-hive.org.au/t/access-esm1-5-release-information/2352) | A coupled global earth system model |

> |[Regional nesting suite (rAM3)](https://forum.access-hive.org.au/t/access-ram3-release-information/4308) | An implementation of the UK Met Office (UKMO) regional nesting suite that supports creating regional atmosphere/land configurations in an Australian context. |

Reviewer: ```suggestion
|[Regional nesting suite (rAM3)](https://forum.access-hive.org.au/t/access-ram3-release-information/4308) | An implementation of the UK Met Office (UKMO) regional nesting suite that supports creating regional atmosphere/land configurations in an Australian context |
```

### Tool
| |  |
| :--- | :---------- |
|[ESMValTool Workflow](https://forum.access-hive.org.au/t/esmvaltool-workflow-releases/1599) | NCI configuration of this tool developed for evaluations of Earth System Models in CMIP |
|[iLAMB workflow](https://forum.access-hive.org.au/t/ilamb-workflow-v1-0-launches-today-here-is-what-you-need-to-know/1600) | NCI configurations of International Land Model Benchmarking (ILAMB)|
|[Benchcab](https://forum.access-hive.org.au/t/benchcab-python-based-software-for-the-evaluation-of-cable/873/11) | Python-based software for the scientific evaluation of CABLE on NCI |

> |[Model Live Diagnostics](https://forum.access-hive.org.au/t/official-model-live-diagnotics-v1-0-released-today-read-on-to-find-out-more/1647) | Framework to check, monitor, visualise and evaluate model behaviour and progress on currently running models on _Gadi_ |

Reviewer: ```suggestion
|[Model Live Diagnostics](https://forum.access-hive.org.au/t/official-model-live-diagnotics-v1-0-released-today-read-on-to-find-out-more/1647) | Framework to check, monitor, visualise and evaluate model behaviour and progress for models currently running on _Gadi_ |
```

> |[ACCESS Intake Catalog](https://forum.access-hive.org.au/t/access-nri-intake-catalog-a-way-to-find-load-and-share-data-on-gadi/1659) | Find load and share ACCESS & ACCESS-related model data on _Gadi_. |

Reviewer: ```suggestion
|[ACCESS Intake Catalog](https://forum.access-hive.org.au/t/access-nri-intake-catalog-a-way-to-find-load-and-share-data-on-gadi/1659) | Find, load and share ACCESS & ACCESS-related model data on _Gadi_ |
```

> |[Payu](https://forum.access-hive.org.au/t/payu-a-workflow-manager-for-some-access-models/1098) | The tool used to run a number of ACCESS models on NCI hardware |

Reviewer: ```suggestion
|[Payu](https://forum.access-hive.org.au/t/payu-a-workflow-manager-for-some-access-models/1098) | Tool used to run a number of ACCESS models on NCI hardware |
```

|[um2nc](https://forum.access-hive.org.au/t/um2nc-a-utility-for-converting-unified-model-files-to-netcdf/3968) | Utility for converting Unified Model (UM) data files to netCDF |

> |[ACCESS-Vis and Visualisation recipes](https://forum.access-hive.org.au/t/access-visualisation-recipes-1-0-0-is-now-available/3970) | Collection of notebooks to enhance the visualisation of ACCESS climate model data using the accessvis python package. |

Reviewer: ```suggestion
|[ACCESS-Vis and Visualisation recipes](https://forum.access-hive.org.au/t/access-visualisation-recipes-1-0-0-is-now-available/3970) | Collection of notebooks to enhance the visualisation of ACCESS climate model data using the `accessvis` python package |
```
